src/main/webapp/index.html

```diff
@@ -25,6 +25,8 @@
         <form onsubmit="return false">
           <p>Enter a YouTube video link:</p>
           <input id="link-input" name="link-input" required pattern="https?://www.youtube.com/watch\?v=.+"/>
+          <p>Enter the maximum number of comments to be analyzed: </p>
+          <input type="number" id="numComments-input" name="numComments-input" min="100" max="1000" value = "100" step="10">
           <small id="url-help" class="form-text text-muted">Format: https://youtube.com/watch?v=...</small>
```

Comment:
The small tag should be above the max number input, since it's referring to the youtube video link